mxcube3/routes/qutils.py

```diff
@@ -1081,15 +1079,18 @@ def init_signals(queue):
     """
     import signals
 
-    for signal in signals.collect_signals:
-        mxcube.collect.connect(mxcube.collect, signal,
-                               signals.task_event_callback)
```

Comment:
Yes it will, because its explicitly connected at for [instance](https://github.com/mxcube/mxcube3/pull/593/files/6e8794a3cebaa651b08a8df8dd34c99c36cc646e#diff-aaa7ddef1045c72f72d678cdfc7b7c84R1091)

They way it was done before was to ambiguous and could potentially lead to clashes depending on the signal name. So Id prefer if we could use a more explicit  way, either like its now or some other way if you have another idea.